### PR TITLE
testing/py-schema: new aport

### PR DESCRIPTION
**Pull request by sgerrand**
💁 https://github.com/keleshev/schema
Schema is a library for validating Python data structures

### CODE REVIEW

**Comment by maxice8** [comment]
Is there a good reason to add a py2- variant ? we'd rather not add new packages for an soon to be EOL language.

**Comment by maxice8** [comment]
ping

**Comment by sgerrand** [comment]
> Is there a good reason to add a py2- variant ?

None really. I'd created a `py-` variant to support my own local use case but there's no need to have it exist here.

**Comment by maxice8** [comment]
Then please keep it only as py3-schema, py2 is soon to be EOL and we are working hard to remove it from everywhere possible.

**Comment by sgerrand** [comment]
:bow: That's done.

File: testing/py3-schema/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+pkgname=py3-schema
+_pkgname=schema
+pkgver=0.6.8
+pkgrel=0
+pkgdesc="Schema is a library for validating Python data structures"
+url="https://github.com/keleshev/$_pkgname"
+arch="noarch"
+license="MIT"
+depends="python"
+makedepends="py-setuptools pytest python2-dev python3-dev"
```
Comment (maxice8) [inline testing/py3-schema/APKBUILD]: can be only py3-setuptools and pytest should be in `checkdepends`

File: testing/py3-schema/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# Maintainer: Sasha Gerrand <alpine-pkgs@sgerrand.com>
+pkgname=py3-schema
+_pkgname=schema
+pkgver=0.6.8
+pkgrel=0
+pkgdesc="Schema is a library for validating Python data structures"
+url="https://github.com/keleshev/$_pkgname"
+arch="noarch"
+license="MIT"
+depends="python"
```
Comment (maxice8) [inline testing/py3-schema/APKBUILD]: should be python3

**Comment by sgerrand** [comment]
♻️

**Pull request closed without merging**